### ISSUE TITLE
Neo4J cannot start with neo4j-framework 3.1.3.46 installed

### ISSUE THREAD

**Issue by rubenst2013** [graphaware/neo4j-framework#69]
Hi,

I installed this plugin [v3.1.3.46] according to the provided instructions and my Neo4J CE 3.1.3 brings me the following failure message:

`Starting Neo4j failed: org.neo4j.server.web.NeoServletContainer-6bbf3ef9@259f8ba9==org.neo4j.server.web.NeoServletContainer,-1,false`

I already tried:
  - A new, empty graphdb directory, freshly initialized
  - Removing all other plugin (I only had apoc so far  and am interested in graphaware/graphgen)
  - Sticking the jar into the Neo4J/plugin directory rather than the one for the graphdb folder
  - all combinations of the above

My environment:
  - windows 10 1703 b15063.138 aka "Creators Update"
  - jre delivered with Neo4J: 1.8.0_60-b23 64bit, mixed mode

Please be aware that my Java skills are quite rusty. :)

I am hoping it's caused just by something I may have missed.
Thanks for any advice on how to fix this and get graphware up and running.

Ruben

**Comment by ikwattro** [comment]
Can you provide more stacktrace ?

**Comment by rubenst2013** [comment]
Hi ikwattro,

I got it up and running by setting the Google stats disablers to false. But when adding the UUID plugin, it again failed. :(

```
2017-04-17 15:11:51.638+0000 INFO  [o.n.s.CommunityNeoServer] ======== Neo4j 3.1.3 ========
2017-04-17 15:11:52.206+0000 INFO  [o.n.s.d.LifecycleManagingDatabase] Starting...
2017-04-17 15:11:55.402+0000 INFO  [o.n.b.v.r.WorkerFactory] Bolt enabled on localhost:7687.
2017-04-17 15:12:13.935+0000 INFO  [o.n.s.d.LifecycleManagingDatabase] Started.
2017-04-17 15:12:14.276+0000 INFO  [o.n.s.m.ThirdPartyJAXRSModule] Mounted unmanaged extension [com.graphaware.server] at [/graphaware]
Apr 17, 2017 5:12:14 PM com.sun.jersey.api.core.PackagesResourceConfig init
INFO: Scanning for root resource and provider classes in the packages:
  com.graphaware.server
Apr 17, 2017 5:12:14 PM com.sun.jersey.server.impl.application.WebApplicationImpl _initiate
INFO: Initiating Jersey application, version 'Jersey: 1.19 02/11/2015 03:25 AM'
Apr 17, 2017 5:12:14 PM com.sun.jersey.server.impl.application.RootResourceUriRules <init>
SEVERE: The ResourceConfig instance does not contain any root resource classes.
2017-04-17 15:12:14.963+0000 ERROR [o.n.s.CommunityNeoServer] Failed to start Neo4j on localhost:7474: org.neo4j.server.web.NeoServletContainer-154a67d1@82fc8ef3==org.neo4j.server.web.NeoServletContainer,-1,false
2017-04-17 15:12:14.963+0000 INFO  [o.n.s.d.LifecycleManagingDatabase] Stopping...
2017-04-17 15:12:15.229+0000 INFO  [o.n.s.d.LifecycleManagingDatabase] Stopped.
```

**Comment by bachmanm** [comment]
should be all fixed in 3.1.3.48. Feel free to use the UUID module for 3.1.3.45 until we release a .48 one
